public/index.html

```diff
@@ -10,6 +10,7 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" href='./index.css'>
```

Comment:
Remove stylesheet link from html. We're importing stylesheets in each related component.